Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -1082,8 +1082,29 @@
       - CarpetPurple
       - CarpetCyan
       - CarpetWhite
+      # Bedsheets
+      - BedsheetBlack
+      - BedsheetBlue
+      - BedsheetBrown
+      - BedsheetGreen
+      - BedsheetGrey
+      - BedsheetOrange
+      - BedsheetPurple
+      - BedsheetRed
+      - BedsheetWhite
+      - BedsheetYellow
+      - BedsheetClown
+      - BedsheetCosmos
+      - BedsheetIan
+      - BedsheetMedical
+      - BedsheetMime
+      - BedsheetNT
+      - BedsheetRainbow
+      - BedsheetBrigmedic
+      - BedsheetUSA
```

Comment:
I'd make the USA one part of the emag list cuz it feels kinda weird for this to be standard Nanotrasen stuff.

Reply:
Actually, in my opinion USA bedsheet is a little bit admeme thing, so I removed recipe